sklearn/feature_extraction/text.py

```diff
@@ -476,6 +480,10 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
         or more alphanumeric characters (punctuation is completely ignored
         and always treated as a token separator).
 
+        If there is a capturing group in token_pattern then the
+        captured group content, not the entire match, becomes the token.
+        For more than one capturing groups, it raises ValueError
```

Comment:
I'm not sure that we need to state this. If we do, it might be "At most one capturing group is permitted."